### PR TITLE
WIP: Add lint stage to CI

### PR DESCRIPTION
**Pull request by yzdann**
Related [discussion](https://groups.google.com/g/prometheus-developers/c/zOnMpZOOdZ4)
* Add lint stage for: flake8 and isort
* Add E731 to flake8 config in tox.in  `do not assign a lambda expression, use a def`
* Clean up W504(line break after binary operator), unused imports, reorder imports and fix indentation
* Remove deprecated `sudo` keyword from `.travis.yml` and also add `os: linux`

### CODE REVIEW

File: prometheus_client/exposition.py [hunk 1]
```diff
@@ -377,10 +378,3 @@ def instance_ip_grouping_key():
     with closing(socket.socket(socket.AF_INET, socket.SOCK_DGRAM)) as s:
         s.connect(('localhost', 0))
         return {'instance': s.getsockname()[0]}
-
-
-try:
-    # Python >3.5 only
```
Comment (brian-brazil) [inline prometheus_client/exposition.py:383]: Why are you removing this?

**Comment by brian-brazil** [comment]
CI is unhappy.

File: prometheus_client/exposition.py (hunk 1 above)
Comment (yzdann) [reply prometheus_client/exposition.py:383]: Sorry, Deleted by mistake with auto flake.

**Comment by yzdann** [comment]
> CI is unhappy.

Changed to WIP again fix and i will mention you again. :pray:

**Comment by brian-brazil** [comment]
Thanks!

**Comment by yzdann** [comment]
Thanks for your review :+1: .

**Pull request merged**